www/docs/further/awesome-trpc.mdx

```diff
@@ -18,6 +18,7 @@ slug: /awesome-trpc
 | Turborepo + Expo + tRPC Starter                                                                  | https://github.com/gunnnnii/turbo-expo-trpc-starter                     |
 | tRPC in a Cloudflare Worker                                                                      | https://github.com/esamattis/trpc-cloudflare-worker                     |
 | tRPC on Commerce Cloud Managed Runtime                                                           | https://github.com/danechitoaie/commerce-cloud-managed-runtime-trpc-poc |
+| tRPC with SvelteKit                                                                              | https://github.com/icflorescu/trpc-sveltekit                            |
```

Comment:
Please move this to "🧩 Extensions & community add-ons" and add your example project here instead :)

Reply:
Feel free to move the extensions section closer to the top too :)